crates/client/devnet/src/predeployed_contracts.rs

```diff
@@ -27,7 +27,7 @@ impl fmt::Display for DevnetKeys {
         writeln!(f, "==== DEVNET PREDEPLOYED CONTRACTS ====")?;
         writeln!(f)?;
         for (i, contract) in self.0.iter().enumerate() {
-            writeln!(f, "(#{}) Address: {:#x}", i + 1, contract.address,)?;
+            writeln!(f, "(#{}) Address: 0x{:0>64}", i + 1, format!("{:x}", contract.address),)?;
```

Comment:
why not use the `to_fixed_hex_string` method?

Reply:
didn't even know it exists - updated, thanks!